src/datasets/log/mod.rs

```diff
@@ -22,7 +22,7 @@ use rand::Rng;
 #[cfg(feature = "rkyv")]
 use rkyv::Archived;
 #[cfg(feature = "wiring")]
-use wiring::prelude::{Unwiring, Wiring};
+use wiring::prelude::{concat_end, concat_mid, concat_start, Unwiring, Wiring};
```

Comment:
These `concat_*` includes aren't necessary since they're attributes available to your derive. It should still compile fine if you remove them (but keep the conditional attributes on the fields).